toolkits/global/packages/global-forms/scss/50-components/_forms.scss

```diff
@@ -136,7 +149,8 @@ label+.c-forms__error,
 }
 
 .c-forms__input--radio,
-.c-forms__input--checkbox {
+.c-forms__input--checkbox,
+.c-form__label--visually-hidden {
```

Comment:
Don't we have a reuseable mixin for that? Or maybe we could even use the associated utiliity class in the partial?

Reply:
@morgaan Good question! And I'm glad you raised this comment. I too remembered about the `u-visually-hidden` mixin. I tried using it in Global Forms but discovered that when you select a radio input (label) the page scrolls to the top (I assume to attempt to scroll the visually hidden element in to view). However, this code used here achieves that same goal but does not cause the page to scroll on user interaction with a form field.

So I have made a note to look into updating the `u-visually-hidden` mixin with this code. Which should be done as it's own PR for Brand Context.

Reply:
Once that has been done, this component can then be updated to use the mixin.

Reply:
👍